backend/routes/download.js

```diff
@@ -240,6 +240,8 @@ router.route('/csv/:showId')
                       studentEmail: `${entryData.studentUsername}@rit.edu`,
```

Comment:
This is the only file that was actually changed, the rest of the PR is just the merged changes from build-error-hotfix